config/openshift/kustomization.yaml

```
@@ -0,0 +1,54 @@
+        app.kubernetes.io/name: validatingwebhookconfiguration
+        app.kubernetes.io/instance: validating-webhook-configuration
+        app.kubernetes.io/component: webhook
+        app.kubernetes.io/created-by: pf-status-relay-operator
+        app.kubernetes.io/part-of: pf-status-relay-operator
+        app.kubernetes.io/managed-by: kustomize
+      annotations:
+        service.beta.openshift.io/inject-cabundle: "true"
+  target:
+    kind: ValidatingWebhookConfiguration
+    name: validating-webhook-configuration
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)